gateway-js/src/__tests__/integration/configuration.test.ts

```diff
@@ -1,16 +1,18 @@
 import gql from 'graphql-tag';
 import http from 'http';
 import mockedEnv from 'mocked-env';
-import { Logger } from 'apollo-server-types';
-import { ApolloGateway } from '../..';
+import {Logger} from 'apollo-server-types';
+import {ApolloGateway} from '../..';
 import {
-  mockSdlQuerySuccess,
-  mockSupergraphSdlRequestSuccess,
   mockApolloConfig,
   mockCloudConfigUrl1,
+  mockCloudConfigUrl2,
+  mockCloudConfigUrl3,
+  mockSdlQuerySuccess,
+  mockSupergraphSdlRequestSuccess,
 } from './nockMocks';
-import { getTestingSupergraphSdl } from '../execution-utils';
-import { MockService } from './networkRequests.test';
+import {getTestingSupergraphSdl} from '../execution-utils';
+import {MockService} from './networkRequests.test';
```

Comment:
Would prefer if we didn't introduce whitespace only changes in code we're not already modifying

Reply:
I'm going to blame that one on my IDE. I cleaned those up and added a CHANGELOG entry in the vNext section.